mapper/thin_edge_json/src/group.rs

```diff
@@ -1,92 +1,195 @@
-use chrono::offset::FixedOffset;
-use chrono::DateTime;
-use std::collections::HashMap;
-
-use crate::measurement::{FlatMeasurementVisitor, GroupedMeasurementVisitor};
-

```

Comment:
I don't know why `cargo fmt` updated this file in its entirety.

The actual changes are:
* The type the timestamp.
* The additional method `is_empty()`.
* The additional tests.

Reply:
Hope the line endings are not CRLF.